packages/create-turbo/src/commands/create/updateCommandsInREADME.js

```diff
@@ -0,0 +1,29 @@
+const fs = require("fs").promises;
+
+const filePath = "../../../../../examples/basic/README.md";
```

Comment:
This won't work when packages and deployed, it needs to use either a root anchored path:
`const rootReadme = path.join(prompts.root, "README.md");

Or, if using `@turbo/workspaces` it can use the `paths.root` on the returned `Project` (this type can be seen [here](https://github.com/vercel/turbo/blob/main/packages/turbo-workspaces/src/types.ts#L24))

Reply:
Done